### PR TITLE
react-react1-week1

### PR DESCRIPTION
**Pull request by negaren**

### CODE REVIEW

File: react/react1/week1/my-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import logo from './logo.svg';
+import './App.css';
+import {MainHearder} from './Header.js';
+import {TasksList} from './Tasks.js';
+
+function App() {
+  return (
```
Comment (senner007) [inline react/react1/week1/my-app/src/App.js:7]: The extra divs seems unnecessary here, imo

File: react/react1/week1/my-app/src/Tasks.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    {
+        description: "Eat breakfast",
+        deadline_date: 'Fri Sep 15 2017',
+    },
+];
+
+export function TasksList() {
+    return (
+        <div class="tasks-main-div">
```
Comment (senner007) [inline react/react1/week1/my-app/src/Tasks.js:20]: Here we need a TaskItem also.
As mentioned in class, I prefer if you separate the logic from the view/jsx
When you inline the mapping in the JSX here, then you make the operation anonymous and less obvious

**Review by senner007 — APPROVED** [review]
👍Good stuff!

File: react/react1/week1/my-app/src/Tasks.js (hunk 2 above)
Comment (negaren) [reply react/react1/week1/my-app/src/Tasks.js]: I appreciate the feedback @senner007 do you approve the merge?

**Comment by negaren** [comment]
@senner007 Kindly check the modified code and please let me know if you approve it

File: react/react1/week1/my-app/src/App.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    <>
+      <MainHearder />
```
Comment (senner007) [inline react/react1/week1/my-app/src/App.js:8]: I think you mean Header here, right?

File: react/react1/week1/my-app/src/Tasks.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        deadline_date: 'Thu Sep 14 2017',
+    },
+    {
+        description: "Eat breakfast",
+        deadline_date: 'Fri Sep 15 2017',
+    },
+];
+
+export function TasksList() {
+    return (
```
Comment (senner007) [inline react/react1/week1/my-app/src/Tasks.js:20]: HEre you can omit the surrounding <></>

File: react/react1/week1/my-app/src/Tasks.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import './Tasks.css';
+import TaskRender from './TaskRender';
+
+const Tasks = [
```
Comment (senner007) [inline react/react1/week1/my-app/src/Tasks.js:4]: Task ought to be camelCase

File: react/react1/week1/my-app/src/TaskRender.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import './Tasks.css';
+
+const TaskRender = ({description, deadline_date}) => {
```
Comment (senner007) [inline react/react1/week1/my-app/src/TaskRender.js:3]: I would name the component according to what it contains. Here it contains a TaskItem

File: react/react1/week1/my-app/src/TaskRender.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+        return (
+            <>
+            <div class="tastks-div">
```
Comment (senner007) [inline react/react1/week1/my-app/src/TaskRender.js:6]: Again you can omit the surrounding tags here. li's inside uls

File: react/react1/week1/my-app/src/App.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+      <ul>
```
Comment (senner007) [inline react/react1/week1/my-app/src/App.js:9]: the <ul></ul> should be inside the TaskList component

**Pull request merged**